dev-haskell/beam-core/beam-core-0.7.2.2-r1.ebuild

```diff
@@ -35,11 +35,7 @@ DEPEND="${RDEPEND}
 		dev-haskell/tasty-hunit )
 "
 
-# I don't see the need to include this build flag
```

Comment:
This was some extra stuff I left commented out in my first revision, in case I needed it later. I don't see the point in keeping it.